examples/simple/main.cc

```diff
@@ -0,0 +1,27 @@
+#include "opentelemetry/sdk/trace/simple_processor.h"
+#include "opentelemetry/sdk/trace/tracer_provider.h"
+#include "opentelemetry/trace/provider.h"
+
+// Using an exporter that simply dumps span data to stdout.
+#include "stdout_exporter.h"
+
+// This models a call to an external library that is instrumented with OT.
+extern void library();
```

Comment:
1. Doesn't need extern.

2. Please move this line to library.h, so it feels more like a cc_library for bazel people.

Reply:
Done.